docs/source/best-practices/performance.md

```diff
@@ -1,210 +0,0 @@
----
```

Comment:
Were these really just floating around in here, unlinked?

In analytics within the last 30-days, this particular doc (`best-practices/performance`) seems to have been getting a _very_ small amount of direct traffic (~200; no referrers), but none of the other `best-practices` guides in this repository have any traffic at all.

Reply:
Yeah they were. There's some old content on schema stitching floating around here too that we should clean up soon.

Once whole query caching is supported again, we're going to do a caching guide that should address everything that was in the `performance` guide.